cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
* I am using OpenMotes
* Only the receiver crashes
* The compilation errors should now be gone - I added `adaptivesec` to the MODULES list of Zoul
* If I used `ccm_star_driver` directly, every call to AES would enable and disable the crypto engine.